old_samples/polygon.html

```diff
@@ -0,0 +1,189 @@
+<!doctype html>
```

Comment:
I'd rather not add to old_samples anymore, did you have some reason?

Reply:
I usually use these samples as first testing. I can remove it.